tests/test_exposure.py

```diff
@@ -825,10 +826,11 @@ def testGetConvexPolygon(self):
 
     def testContainsSkyCoords(self):
         """Test the sky coord containment code."""
-        self.assertRaises(ValueError,
-                          self.exposureMiOnly.containsSkyCoords,
-                          0.0,
-                          0.0)
+        self.assertRaisesRegex(ValueError,
```

Comment:
Probably not worth changing for this simple case, but I always do assertRaises in a context manager so that the call signature is completely obvious:

```
with self.assertRaisesRegex(ValueError, msg):
    self.exposureMiOnly.containsSkyCoords(0, 0)
```